### ISSUE TITLE
Prevent bookmarks from overwritten

### ISSUE THREAD

**Issue by woaidangyang** [ahrm/sioyek#335]
If I write a new bookmark without changing the location, the previous bookmark is replaced. Since I use bookmarks as notes, this behavior makes me worried that I would lose some of my notes.

Thanks!

**Comment by ahrm** [comment]
> If I write a new bookmark without changing the location, the previous bookmark is replaced. Since I use bookmarks as notes, this behavior makes me worried that I would lose some of my notes.

That's not true (or at least it shouldn't be). We don't overwrite bookmarks if they have the same location. Are you experiencing different behaviour?

**Comment by woaidangyang** [comment]
I tested it again and found it doesn't overwrite bookmarks. Sorry!
